### ISSUE TITLE
When can we expect the next release?

### ISSUE THREAD

**Issue by rogerluan** [TextureGroup/Texture#1787]
Hello!

First of all, thanks for this amazing framework. It makes our apps so much smoother!

Our team has been waiting for 2 specific fixes that have already been merged but oddly enough haven't been released yet. So I was wondering:

1. What's the rationale for you guys to merge a PR, if it's not followed by a release?
1. When can we expect the next release?

The two PRs we're waiting for are the following:

TextureGroup/Texture#1405  
TextureGroup/Texture#1579

I appreciate any feedback that we can get.

**Comment by sgrgrsn** [comment]
6 months ago @rahul-malik [mentioned](https://github.com/TextureGroup/Texture/issues/1688#issuecomment-535942867) that we could expect a new release soon, but unfortunately there has been no new release since the 2.8.1 which is 7 months ago.

I've also tried so reach out to ask if there was something we could help with to get the new release out faster, but I haven't heard anything 👎

**Comment by rogerluan** [comment]
Thanks for the input @sgrgrsn !

I guess we'll have to patiently wait, I hope it can be soon enough 🙏

**Comment by rogerluan** [comment]
Still no update 😞

**Comment by garrettmoon** [comment]
Hey y'all, sorry for the radio silence. We'll get out a new release next month.

**Comment by sgrgrsn** [comment]
> Hey y'all, sorry for the radio silence. We'll get out a new release next month.

That sounds really good! Let's us know if there's anything we can do to speed up the process.

**Comment by rogerluan** [comment]
@garrettmoon first of all thanks for the update! 🚀 

I truly don't mean to rush you since you already gave us an ETA, but I was wondering what's delaying this release, if the features/fixes we're looking for are already in master? Shouldn't a release require simply updating the podspec and pushing the pod?

I'm asking just out of curiosity really, I appreciate the efforts of the entire TextureGroup team! 🙌

**Comment by huangjianke** [comment]
> Hey y'all, sorry for the radio silence. We'll get out a new release next month.

@garrettmoon  Still no update 😞

**Comment by garrettmoon** [comment]
@huangjianke we're still on track to get a release out this month as promised!

**Comment by huangjianke** [comment]
> @huangjianke we're still on track to get a release out this month as promised!

Great, Thanks

**Comment by rogerluan** [comment]
Guys, there's a RC out 🚀 https://github.com/TextureGroup/Texture/releases/tag/3.0.0-rc.1

Thanks @garrettmoon ! Is there anything specific you want us to look for, when testing? Or do we need a v3.0.0 migration checklist, or guide?

**Comment by garrettmoon** [comment]
Hey y'all, as promised we're working towards a 3.0 release and have a release candidate out to CocoaPods (RC2).

@rogerluan A 3.0 migration guide is a great idea! [I put up a starter doc which I'd love folks to add tips to.](https://github.com/TextureGroup/Texture/blob/master/ThreeMigrationGuide.md)

**Comment by rogerluan** [comment]
Thanks for creating the doc! I added my entry to the only breaking change that prevented my project from compiling once I upgraded: https://github.com/TextureGroup/Texture/pull/1878

We were also applying 2 changes that were present in past PRs that got merged and released in v3.0, so not we're not applying those changes anymore. That's gif image support, and an issue with fonts defaulting to Times New Roman. I thought it didn't make sense to mention that in the doc, though.
